test/manual/amp-consent/amp-consent-granular-consent.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
This file has a large diff. Making sure that other than removing the `AMP.toggleExperiment` block, the rest is just lint/prettier changes?

Reply:
Correct, formatting but also removing a script for `amp-ad`, which is not used.